core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/DataFrame.kt

```diff
@@ -135,6 +135,8 @@ public operator fun <T> DataFrame<T>.get(columnRange: ClosedRange<String>): Data
 
 internal val ColumnsContainer<*>.ncol get() = columnsCount()
 internal val AnyFrame.nrow get() = rowsCount()
+
+@PublishedApi
 internal val AnyFrame.indices get() = indices()
```

Comment:
inline functions that use this internal properties can use already public functions instead